utils/test_harness/sysman/src/test_harness_sysman_device.cpp

```diff
@@ -30,6 +30,27 @@ get_sysman_device_properties(zes_device_handle_t device) {
   return properties;
 }
 
+std::vector<zes_subdevice_exp_properties_t>
+get_sysman_subdevice_properties(zes_device_handle_t device, uint32_t *pCount){
+  std::vector<zes_subdevice_exp_properties_t> subDeviceProperties(*pCount);
+  if (*pCount == 0) {
+    EXPECT_EQ(ZE_RESULT_SUCCESS ,zesDeviceGetSubDevicePropertiesExp(device, pCount, nullptr));
+  }
```

Comment:
Looks like code formatting is missing. Take Vishnu's help regarding this.

Reply:
Done.